go.mod

```diff
@@ -41,8 +43,11 @@ require (
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/mitchellh/mapstructure v1.5.1-0.20231216201459-8508981c8b6c
 	github.com/oklog/ulid v1.3.1
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/elasticsearchexporter v0.99.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.98.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/processor/attributesprocessor v0.98.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor v0.99.0
```

Comment:
there will be a conflict: https://github.com/elastic/elastic-agent/pull/4638

Reply:
Let's wait to get #4638 merged first since that's a more important change IMO and then I'll rebase this PR on `main`.